app/controllers/pages_controller.rb

```diff
@@ -23,4 +26,10 @@ def terms
   def privacy
     skip_authorization
   end
+
+  private
+
+    def marketing_site?
+      Rails.env.production? && request.host.start_with?('www.')
```

Comment:
switching things by environment here is bad practice, maybe use a flag in rad_common.yml instead